.github/workflows/release.yml

```diff
@@ -21,7 +21,7 @@ jobs:
         run: |
           make html
       - name: deploy
-        uses: peaceiris/actions-gh-pages@v3
+        uses: peaceiris/actions-gh-pages@286809f  # v3.5.8
```

Comment:
To prove this syntax works I setup a demo here - https://github.com/oliver-sanders/cylc-sphinx-extensions/pull/3